### PR TITLE
Added Sieve Of Eratosthenes [C] and minor changes in PR Template

### PR DESCRIPTION
**Pull request by aashutoshrathi**
#### By submitting this pull request I confirm I've read and complied with the below requirements.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm name} [{Language}]`, not `Update README.md` or `Added new code`.

### CODE REVIEW

**Comment by singhpratyush** [comment]
@aashutoshrathi: Please create two different pull requests for the two issues.

File: sieve_of _eratosthenes/SieveOfEratosthenes.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	bool numbers[n+1];
+	int i,sqrt_n = sqrt(n)+1,j;
+	
+	for (i = 0; i < n+1; i++){
+		numbers[i] = true;      // assigning every number "true" 
+	}
+
+	for (i = 2; i < sqrt_n; i++){
+		if (numbers[i] == true){
+
```
Comment (aviaryan) [inline sieve_of _eratosthenes/SieveOfEratosthenes.c]: Unneeded blank line here.

File: sieve_of _eratosthenes/SieveOfEratosthenes.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+#include <math.h>     // for using square root 
+#include <stdbool.h>  // to include bool datatype 
+
+void sieveOfEratosthenes(int n){
+	bool numbers[n+1];
+	int i,sqrt_n = sqrt(n)+1,j;
```
Comment (aviaryan) [inline sieve_of _eratosthenes/SieveOfEratosthenes.c]: Space after comma (`,`) is preferred.

File: sieve_of _eratosthenes/SieveOfEratosthenes.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	
+	for (i = 0; i < n+1; i++){
+		numbers[i] = true;      // assigning every number "true" 
+	}
+
+	for (i = 2; i < sqrt_n; i++){
+		if (numbers[i] == true){
+
+			for (j = 2*i; j < n+1; j=j+i){
```
Comment (aviaryan) [inline sieve_of _eratosthenes/SieveOfEratosthenes.c]: Please follow a consistent style. You seem to be preferring space before and after assignment/logical operators.  Eg `j = 2*i`. So make it `j = j+i`.

Comment (aviaryan) [inline sieve_of _eratosthenes/SieveOfEratosthenes.c]: Please check other places for this issue as well.

**Comment by aashutoshrathi** [comment]
@singhpratyush should i close this pr and create a new one which only add Sieve Of Eratosthenes [C]

**Comment by singhpratyush** [comment]
I think that would be the better way. @aviaryan : Views?

**Pull request closed without merging**